common/lib/xmodule/xmodule/video_module/video_module.py

```diff
@@ -326,7 +326,10 @@ def get_html(self):
             'showCaptions': json.dumps(self.show_captions),
             'generalSpeed': self.global_speed,
             'speed': self.speed,
-            'autoAdvance': self.auto_advance or getattr(self, 'video_auto_advance', False),
+            'autoAdvance':
+                self.auto_advance or
+                getattr(self, 'video_auto_advance', None) or
+                settings.FEATURES.get('AUTOADVANCE_VIDEOS', False),
```

Comment:
I don't think this is what @sstack22 is asking for - the feature flag shouldn't turn auto-advance on for all courses all the time when set to True. We'll always want to be able to choose which specific courses having it enabled. Instead, when set to False, it should disable the feature completely across all courses; when it's enabled, people can choose to enable it on individual courses.

So it should actually be ANDing with the existing logic, not ORing.